Contract Template/admin/initServer.sqf

```diff
@@ -1,5 +1,15 @@
 // Do not Edit this file.
 
+// Mathematical Heresy
+systemTimeUTC params ["_hour", "_minute", "_second"];
+private _timeUntilStart = (14 * 60 * 60) - (_hour * 60 * 60 + _minute * 60 + _second); // start time (1400z) - current time = time until start time
```

Comment:
```suggestion
systemTimeUTC params ["", "", "", "_hour", "_minute"];
private _timeUntilStart = ((14 * 60) - (_hour * 60 + _minute)) * 60; // start time (1400z) - current time = time until start time
```
Fix `systemTimeUTC` parameters and use minute-accurate time instead of second-accurate, we really don't need to be second-accurate IMO (but it doesn't hurt if you want to be).